lib/utils/SchemaBuilderXSD.js

```diff
@@ -681,7 +682,7 @@ class SchemaBuilderXSD {
       wsdlElement.pattern = element.pattern;
       wsdlElement.enum = element.enum;
 
-      this.getChildren(element, wsdlElement, elements);
+      this.getChildren(getDeepCopyOfObject(element), wsdlElement, elements, targetNamespace);
```

Comment:
@VShingala I had removed this `getDeepCopyOfObject` from here as part of https://github.com/postmanlabs/wsdl-to-postman/pull/196, and it got added back again, I believe, due to resolving conflicts. We should remove this and test it.